conf_sp_idp/idp/idp.py

```diff
@@ -1120,12 +1127,15 @@ def application(environ, start_response):
         pass
     ds.DefaultSignature(sign_alg, digest_alg)
 
-    SRV = WSGIServer((HOST, PORT), application)
-
+    ssl_context = None
     _https = ""
-    if CONFIG.HTTPS:
+    if CONFIG.HTTPS:        
```

Comment:
```suggestion
    if CONFIG.HTTPS:
```

There is some trailing whitespace that probably shouldn't be there.